### DISCUSSION TITLE
Introducing OpenVINO.NET: A High Quality .NET Wrapper for OpenVINO

### DISCUSSION THREAD

**Discussion by sdcb** [openvinotoolkit/openvino#20379] [Show and tell]
Hello everyone!

We all know how cool OpenVINO is, don't we? Its power and flexibility in handling deep learning tasks across multiple languages like C, C++, and Python is commendable. But for those of us who are C# enthusiasts, we've noticed a gap... until now!

I have spent my holidays working hard to address this very gap, and I'm thrilled to introduce to you: [OpenVINO.NET](https://github.com/sdcb/OpenVINO.NET) - a high-quality, open source .NET wrapper for OpenVINO.

OpenVINO.NET supports all C APIs of OpenVINO through CppSharp and provides a clean and user-friendly C# interface. Every interface is meticulously unit-tested and is accompanied by comprehensive XML documentation.

One of the highlights is the native support for the OpenVINO wrapper based on PaddleOCR. According to tests, OpenVINO.NET has delivered performance gains that exceed PaddleSharp by over 400%.

Plus, it's cross-platform compatible. I have provided NuGet packages for eight different platforms, officially supported.

I've poured a lot of effort and time into this project and I would be delighted for you to try it out and see the benefits for yourself. If you like the project and find it useful, I encourage you to give it a star on GitHub: [OpenVINO.NET](https://github.com/sdcb/OpenVINO.NET)

I'm keen to hear your feedback and suggestions. Please feel free to contribute to the project or connect with me if you have any questions or concerns.

Thank you and happy coding!

Best regards,

sdcb

**Comment by devangaggarwal** [comment]
Great project! 👍 This is very useful for .NET developers. Thank you for your great work in building this project!
